src/elements/Button/Button.styl

```diff
@@ -123,17 +134,19 @@
   transition-duration $duration-shortest
 
   // Children
-
   & > .content
-    padding $space-normal
-
+    padding $space-zero
+    padding-bottom 7px
```

Comment:
We do not encourage hardcoding paddings. If some centering has to be done, we would prefer
```css
.class {
  display: flex;
  align-items: center;
  justify-content: center;
}
```